README.md

```diff
@@ -7,5 +7,5 @@ This package is a Twig extension that provides the following:
 
  * [`html_to_markdown`][2] filter: generates Markdown from an HTML block.
 
-[1]: https://twig.symfony.com/markdown_to_html
-[2]: https://twig.symfony.com/html_to_markdown
+[1]: https://twig.symfony.com/doc/2.x/filters/markdown_to_html.html
```

Comment:
`2.x/` is not required: https://twig.symfony.com/doc/filters/markdown_to_html.html works.

Reply:
Just deleted doc version